docs/SRS.md

```diff
@@ -111,81 +132,134 @@ Deployment:
 TBD
 
 Testing:
+
 - Cucumber
 - JUnit
 
 ## 3. Specific Requirements
 
 ### 3.1 Functionality
 
-This section will explain the different use cases, you could see in the Use Case Diagram, and their functionality.  
-Until December, we plan to implement:
+This section will explain the different use cases, you could see in the Use Case Diagram, and their
+functionality.  
+Until December we plan to implement:
+
+This section will explain the different use cases, you could see in the Use Case Diagram, and their
+functionality.  
+Until December we plan to implement:
+
+- 3.1.1 Create Account
+- 3.1.1 Logging in
+- 3.1.2 Logging out
+- 3.1.3 Manual adding of data
+- 3.1.4 Scan receipts
+
+#### 3.1.1 Create account
+
+To identify all users we need an account system. This account system enables us to build important
+functions such as sharing costs or a personalized overview over finances (Keeping track of your
+finances).
+
+#### 3.1.2 Logging in
+
+The app will provide the possibility to register and log in. This will also make the usability
+easier when a user wants to manage his expenses because they don't have to enter their mail address
+every time.
+
+#### 3.1.3 Logging out
+
+In case you share your phone, have multiple accounts or just want to be cautious about your privacy
+you should be able to manually log out
+
+#### 3.1.4 Manual adding of data
 
-> TBD
+This is one of the essentials of our project. The user gets the possibility to manually add his
+expenses. Therefore the User has tho enter the store of the purchase, and the items with the
+respective deposited category and the price
```

Comment:
As a follow-up ticket, we should create Use-Case Markdowns, where we can also embed the use-case diagram :)